assignment3.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang ="en">
+    <head>
+        <!-- Lucas Sibbernsen
+            default.html
+            INFO2439
+            Thoendel
+            fall 2019
+        -->
+        <title>Lucas Sibbernsen | INFO 2439 | Assignment 3</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css?family=Tangerine">
+        <style>
+    body {
+      font-family: 'Tangerine', serif;
+      font-size: 48px;
+    }
+  </style>
+        <link rel="stylesheet" href="_css/sibbernsen_a3.css"/>     
```

Comment:
Something isn't jiving with your css links...if you ever want to dig in a sort out what's going on here - flag me down.